app/admin/custom-forms/index/route.js

```diff
@@ -25,7 +25,7 @@ export default AbstractIndexRoute.extend(ModalHelper, UserSession, {
   actions: {
     deleteItem(item) {
       let i18n = this.get('i18n');
-      let message = i18n.t('admin.customForms.messages.deleteForm');
+      let message = i18n.t('messages.delete_singular', { name: 'form' });
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.